privacyidea/api/policy.py

```diff
@@ -523,9 +523,9 @@ def get_policy_defs(scope=None):
     dynamic_pol = get_dynamic_policy_definitions()
 
     # combine static and dynamic policies
-    keys = static_pol.keys() + dynamic_pol.keys()
-    pol = {k: dict(static_pol.get(k, {}).items()
-                   + dynamic_pol.get(k, {}).items()) for k in keys}
+    keys = list(static_pol.keys()) + list(dynamic_pol.keys())
```

Comment:
Wouldn't it be

~~~~python
keys = list(static_pol) + list(dynamic_pol)
~~~~

Reply:
Yes, You are right.

Reply:
Done